.github/workflows/integration-test-helm-suite.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        kubernetes-versions : ['v1.17.17', 'v1.23.0']
+        kubernetes-versions : ['v1.17.17','v1.18.0','v1.19.0','v1.20.0','v1.21.0','v1.22.0', 'v1.23.0']
```

Comment:
Are these changes temporary for this PR?

Reply:
Yes @BlaineEXE that is correct. will remove it after testing.